src/runtime/runtime.js

```diff
@@ -84,43 +87,46 @@ export function getRuntime() {
 }
 
 /**
+ * Installs SwG runtime.
  * @param {!Window} win
  */
 export function installRuntime(win) {
+  // Only install the SwG runtime once.
   if (win[RUNTIME_PROP] && !isArray(win[RUNTIME_PROP])) {
     return;
   }
 
+  // Create a SwG runtime.
   const runtime = new Runtime(win);
 
-  const waitingArray = [].concat(win[RUNTIME_PROP], win[RUNTIME_LEGACY_PROP]);
-
-  // Public runtime.
+  // Create a public version of the SwG runtime.
   const publicRuntime = createPublicRuntime(runtime);
 
-  const dependencyInstaller = {};
-
   /**
+   * Executes a callback when SwG runtime is ready.
    * @param {function(!Subscriptions)} callback
    */
-  function pushDependency(callback) {
+  function push(callback) {
```

Comment:
Nit: naming is confusing since there is an `Array.push` already.  More so since it's not actually pushing anything to an array.

Reply:
How about the name `callWhenRuntimeIsReady`?

Reply:
yes